Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -77,6 +77,9 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath, b
 
 if ~xASL_exist(PWI_Path, 'file')
     warning('Skipped xASL_wrp_Quantify: files missing, please rerun step 4: xASL_wrp_ResampleASL');
+	if bUseBasilQuantification
+		fprintf('%s\n', 'Note the PWI4D.nii used in BASIL is deleted after quantification');
```

Comment:
@jan-petr I believe the `PWI4D` is now also used by the default quantification, for generalization.

Reply:
It is not - I have tried that - BASIL quantification did not work and normal quantification with PWI.nii worked when I re-run the normal quant only...

Reply:
OK